src/Spryker/Style/SprykerStyle.php

```diff
@@ -124,7 +124,7 @@ public function endInstall(StageInterface $stage)
     public function startSection(SectionInterface $section)
     {
         $this->timer->start($section);
-        $message = sprintf('<bg=green;options=bold> Section %s</>', $section->getName());
+        $message = sprintf('<fg=blue;bg=green;options=bold> Section %s</>', $section->getName());
```

Comment:
For me this change looks not nice on black background. I tried with foreground black and not bold, this seems to look good on black and white background.

`$message = sprintf('<fg=black;bg=green> Section %s</>', $section->getName());`

I also checked the current color scheme and it also looks good to me on black and white backgrounds. I think there will be always one who has not a perfect coloring.

Reply:
1) i am working on dark theme
2) you using bright colour (or white-ty not sure how to translate correctly) on bright colour - it is never readable.
3) check screenshots

Reply:
For me white on green looks good at least much better than blue on green.

You can drag and drop images into github comments

Reply:
This is how it looks for me:

![image 2017-12-13 at 10 35 21 am 0q242c2b3b2p](https://user-images.githubusercontent.com/1382877/33931856-9cab4dfa-dff1-11e7-851f-d6c722fd91b6.png)

Reply:
looks like i know problem.
Different colour scheme at night.
At night green became more yellow - so it change to less readable.

Reply:
Why not using black on colored backgrounds always?

Reply:
Before we go all https://www.splitbrain.org/blog/2008-09/18-calculating_color_contrast_with_php here :D

Reply:
**WWW**:
1) why it is using background colors at all? or at least make `style` configurable…
2) why `Symfony\Component\Console\Helper\ProgressBar` is not used? (it can be styled: https://stillat.com/blog/2016/12/07/laravel-artisan-custom-styles-custom-progress-bar-styles)
3) why `Symfony\Component\Console\Helper\Table` for summary is not used? (good for the whole summary of installation: https://stillat.com/blog/2016/12/03/custom-command-styles-with-laravel-artisan)

Reply:
1. To visualize the sections
2. It is already used
3. There is no summary all commands print there output in different verbosity levels.

You can open a PR with how you would like to have it.